mysql-test/suite/tokudb_rpl/r/rpl_extra_col_master_tokudb.result

```diff
@@ -429,6 +429,8 @@ include/check_slave_is_running.inc
 ** Stop and Reset Slave **
 
 STOP SLAVE;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
Explained above. The waring comes from rpl_tests/rpl_extra_col_master.test